packages/ui/v2/core/apps/AllApps.tsx

```diff
@@ -86,8 +86,8 @@ export default function AllApps({ apps }: AllAppsPropsType) {
               router.replace(router.asPath.split("?")[0], undefined, { shallow: true });
             }}
             className={classNames(
-              selectedCategory === null ? "bg-gray-200" : "bg-gray-50",
-              "rounded-md px-4 py-2.5 text-sm font-medium text-gray-900 hover:cursor-pointer hover:bg-gray-200"
+              selectedCategory === null ? "bg-gray-900 text-gray-50" : "bg-gray-50 text-gray-900",
+              "rounded-md px-4 py-2.5 text-sm font-medium hover:cursor-pointer hover:bg-gray-900 hover:text-gray-50"
```

Comment:
Some tweaks to the categories strip in the App Store, to have a black background on selected or hover.